### PR TITLE
Added Missed recording label

### PR DESCRIPTION
**Pull request by clare**

### CODE REVIEW

File: src/components/Video/RecordingControls.vue [hunk 1]
```diff
@@ -36,6 +36,15 @@
                 Missed track
               </b-dropdown-item>
 
+              <b-dropdown-item
+                v-b-tooltip.hover.left="
+                  'Missing an earler recording that explains how the animal got to where it is now'
```
Comment (gferraro) [inline src/components/Video/RecordingControls.vue:41]: earlier

File: src/components/Video/RecordingControls.vue [hunk 2]
```diff
@@ -36,6 +36,15 @@
...
+                "
+                @click="addTag('missed recording')"
+              >
+                Missed reccording
```
Comment (hardiesoft) [inline src/components/Video/RecordingControls.vue:45]: Typo `reccording`

**Review by hardiesoft — APPROVED** [review]
Just a small typo to fix

**Review by gferraro — APPROVED** [review]
Cool

**Pull request merged**